src/dotnet-interactive-vscode/common/interfaces/contracts.ts

```diff
@@ -335,6 +335,7 @@ export interface ResolvedPackageReference extends PackageReference {
     assemblyPaths: Array<string>;
     probingPaths: Array<string>;
     packageRoot: string;
+    requested: boolean;
```

Comment:
We should avoid modifying this contract. If we need this property as an implementation detail, we can leave the contract alone by making the property internal.

Reply:
Okay makes sense.  So long as t's never serialized right?

Reply:
removed property